### DISCUSSION TITLE
Patch request is invalid -message

### DISCUSSION THREAD

**Discussion by mmedjane** [PrestaShop/PrestaShop#33141] [Q&A]
Hello everyone :

When I try to make a PATCH request according to the documentation, I get an error 23 saying the patch method is not valid (see screenshot below): 

[
![image](https://github.com/PrestaShop/PrestaShop/assets/131868693/ee3eb393-b8be-49c3-a2ad-c5dc7a78ba96)
](url)

 I really wonder why considering that I followed precisely what the instructions said (meaning adding the id in the request and just the info I try to modify) : 
[
![image](https://github.com/PrestaShop/PrestaShop/assets/131868693/a52ee6d1-3242-4bc8-abc1-85fef1cd3637)
](url)

So is it me or is there a larger problem ? 
Thank you in advance !

**Comment by kpodemski** [comment]
Hello @mmedjane 

What's your PrestaShop version?

**Reply by mmedjane** [reply]
Hello ! 
Hello, 
So I have the 1.7.8.9 version of prestashop installed.

**Reply by kpodemski** [reply]
The PATCH method has been added in PrestaShop 8

**Reply by mmedjane** [reply]
thank you very much, that was it

**Reply by waflessnet** [reply]
What is the method then to update the stock in 1.7.X.X?

**Reply by jmdv-es** [reply]
> What is the method then to update the stock in 1.7.X.X?

Looks like PUT is the only option... right?
